packages/dd-trace/src/log/index.js

```diff
@@ -111,11 +111,11 @@ const log = {
 
   isEnabled (fleetStableConfigValue, localStableConfigValue) {
     return isTrue(
-      (fleetStableConfigValue ??
+      fleetStableConfigValue ??
       getEnvironmentVariable('DD_TRACE_DEBUG') ??
-      getEnvironmentVariable('OTEL_LOG_LEVEL') === 'debug') ||
-      (localStableConfigValue ??
-      config.enabled)
+      (getEnvironmentVariable('OTEL_LOG_LEVEL') === 'debug' || undefined) ??
```

Comment:
This also slipped through earlier. The change would have been caused in case the first three parts would be falsy. In that case the other two values should not be checked anymore. That is now the case again.

Reply:
`(a || undefined) ?? b` isn't equivalent to `a || b` when `a` is guaranteed boolean?